### ISSUE TITLE
Basic information about source language not visible, and sound not triggerable after popup click

### ISSUE THREAD

**Issue by Braxton1981** [sienori/simple-translate#530]
Would be nice to have information about what language the extension translated after selecting the text and pressing the popup button. I mean: the translation is good, but there is no way we can know what language the source text was written it. A label saying, for example "From: Russian" would suffice. 

Also, after selecting a text in some other language and clicking the popup button that your extension provides, there is no option to "play it with voice" so we cannot hear the source text. If we want to play it, we have to go to the upper Toolbar, with the source text selected, and click the icon of the extension.

Would be really, really nice features to implement, for this extension to feel complete.
Thanks!

**Comment by trueauracoral** [comment]
Yes actually being able to see what language was translated from would be the best
